### ISSUE TITLE
An option to display some simple card shadow

### ISSUE THREAD

**Issue by jan-kleks** [KyleU/solitaire.gg#40]

**Comment by KyleU** [comment]
Good idea, I tried to implement that a while ago. Dragging a stack of cards kept stacking the shadows. I'll take another swing at it, thanks.

**Comment by jan-kleks** [comment]
One more thing about this screensho,t this time regarding shadows, if you look closely you can notice that card shadows are displayed all the time (and not only when moving the cards) it seems to add an interesting sense of depth to cards and to the whole layout:
![image](https://cloud.githubusercontent.com/assets/15075571/25464382/67547348-2afc-11e7-8f51-ddbe01e78e46.png)

**Comment by KyleU** [comment]
Yeah, it definitely looks better with shadows. But if I just add it to the card images, shadows "stack", making overlapped shadows darker. I played with a few ideas, none of them both fast and pretty.
